actions.py

```diff
@@ -0,0 +1,220 @@
+import logging
+import random
+
+import card as c
+from datetime import datetime
+
+from telegram import Message, Chat
+
+from bot_config import TIME_REMOVAL_AFTER_SKIP
+from errors import DeckEmptyError, NotEnoughPlayersError
+from internationalization import __, _
+from shared_vars import gm, botan
+from user_setting import UserSetting
+from utils import send_async, display_name, game_is_running
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+    ,filename='log.log'
+)
+logger = logging.getLogger(__name__)
+
+
+class Countdown(object):
+    player = None
+    job_queue = None
+
+    def __init__(self, player, job_queue):
+        self.player = player
+        self.job_queue = job_queue
+
+
+# TODO do_skip() could get executed in another thread (it can be a job), so it looks like it can't use game.translate?
```

Comment:
Do you mean the `_` or `__` function fails? Can you post the error that happens?

Reply:
Just did a quick test, adding a __ function inside do_skip and letting it trigger in a fast game:
https://i.gyazo.com/3b95384bfdf04d4d62ce6b44e1430f6c.png

Here's the text trace:
2017-10-23 19:08:17,163 - utils - ERROR - Message text is empty
Traceback (most recent call last):
  File "F:\PROJECTS\mau_mau_bot\utils.py", line 92, in send_async
    bot.sendMessage(*args, **kwargs)
  File "C:\Users\Jarv\AppData\Local\Programs\Python\Python36-32\lib\site-packages\telegram\bot.py", line 57, in decorator
    result = func(self, *args, **kwargs)
  File "C:\Users\Jarv\AppData\Local\Programs\Python\Python36-32\lib\site-packages\telegram\bot.py", line 69, in decorator
    return self._message_wrapper(url, data, *args, **kwargs)
  File "C:\Users\Jarv\AppData\Local\Programs\Python\Python36-32\lib\site-packages\telegram\bot.py", line 165, in _message_wrapper
    result = self._request.post(url, data, timeout=kwargs.get('timeout'))
  File "C:\Users\Jarv\AppData\Local\Programs\Python\Python36-32\lib\site-packages\telegram\utils\request.py", line 271, in post
    **urlopen_kwargs)
  File "C:\Users\Jarv\AppData\Local\Programs\Python\Python36-32\lib\site-packages\telegram\utils\request.py", line 209, in _request_wrapper
    raise BadRequest(message)
telegram.error.BadRequest: Message text is empty